front/src/routes/integration/all/zigbee2mqtt/setup-page/actions.js

```diff
@@ -0,0 +1,171 @@
+import createActionsIntegration from '../../../../../actions/integration';
+import { RequestStatus } from '../../../../../utils/consts';
+import dayjs from 'dayjs';
+import relativeTime from 'dayjs/plugin/relativeTime';
+import get from 'get-value';
+
+dayjs.extend(relativeTime);
+
+const createActions = store => {
+  const integrationActions = createActionsIntegration(store);
+  const actions = {
+    async checkStatus(state) {
+      let zigbee2mqttStatus = {
+        usbConfigured: false,
+        mqttExist: false,
+        mqttRunning: false,
+        zigbee2mqttExist: false,
+        zigbee2mqttRunning: false,
+        gladysConnected: false,
+        zigbee2mqttConnected: false,
+        z2mEnabled: false,
+        dockerBased: false,
+        networkModeValid: false
+      };
+      try {
+        zigbee2mqttStatus = await state.httpClient.get('/api/v1/service/zigbee2mqtt/status');
+      } finally {
+        store.setState({
+          usbConfigured: zigbee2mqttStatus.usbConfigured,
+          mqttExist: zigbee2mqttStatus.mqttExist,
+          mqttRunning: zigbee2mqttStatus.mqttRunning,
+          zigbee2mqttExist: zigbee2mqttStatus.zigbee2mqttExist,
+          zigbee2mqttRunning: zigbee2mqttStatus.zigbee2mqttRunning,
+          gladysConnected: zigbee2mqttStatus.gladysConnected,
+          zigbee2mqttConnected: zigbee2mqttStatus.zigbee2mqttConnected,
+          z2mEnabled: zigbee2mqttStatus.z2mEnabled,
+          dockerBased: zigbee2mqttStatus.dockerBased,
+          networkModeValid: zigbee2mqttStatus.networkModeValid
+        });
+      }
+    },
+    async getContainers(state) {
+      let mqtt4z2mContainerExists = false;
+      let z2mContainerExists = false;
+      let mqtt4z2mContainerRunning = false;
+      let z2mContainerRunning = false;
+      //      let z2mEnabled = state.z2mEnabled;
+      let dockerContainers = [];
+      store.setState({
+        DockerGetContainersStatus: RequestStatus.Getting
+      });
+      try {
+        const containers = await state.httpClient.get('/api/v1/system/container');
+        containers.forEach(container => {
+          container.created_at_formatted = dayjs(container.created_at * 1000)
+            .locale(state.user.language)
+            .fromNow();
+          if (container.name === '/gladys-z2m-zigbee2mqtt') {
```

Comment:
I'm not sur we want this logic at front side. Maybe the /api/v1/service/zigbee2mqtt/status can compute all of this in backend?

Reply:
Removed (it's already in status)